plugins/Transitions/API.php

```diff
@@ -55,20 +55,23 @@ public function getTransitionsForPageUrl($pageUrl, $idSite, $period, $date, $seg
      * @param $period
      * @param $date
      * @param bool $segment
-     * @param bool $limitBeforeGrouping
+     * @param int $limitBeforeGrouping
      * @param string $parts
      * @return array
      * @throws Exception
      */
     public function getTransitionsForAction($actionName, $actionType, $idSite, $period, $date,
-                                            $segment = false, $limitBeforeGrouping = false, $parts = 'all')
+                                            $segment = false, $limitBeforeGrouping = 0, $parts = 'all')
     {
         Piwik::checkUserHasViewAccess($idSite);
 
         if (!$this->isPeriodAllowed($idSite, $period, $date)) {
             throw new Exception('PeriodNotAllowed');
         }
 
+        if ($limitBeforeGrouping && !is_numeric($limitBeforeGrouping)) {
+            throw new Exception('LimitBeforeGroupingIsInvalid');
```

Comment:
I don't think it has any worth to have this exception message translatable. The message should only occur when someone calls the api method manually, as our UI should always use correct values.